masterlist.yaml

```
@@ -11411,6 +11411,24 @@ plugins:
       - crc: 0x2AFC6CDF
         util: 'TES4Edit v4.0.4'
 
+  - name: 'Tomb of the Unknown Knight.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/6935/' ]
+    msg:
+      - <<: *wildEdits
+        subs: [ 'Remove **00033569**, **000CA158**, **00015C3D**, & **00006622** from **Tomb of the Unknown Knight.esp**.' ]
```

None of these records appear to be edited by the mod, but I assume you meant **00014EB3**, **00093CC8**, **0012DD1C**, & **0014C46A** since the checksum matches after deleting them.

```suggestion
        subs: [ 'Remove **00014EB3**, **00093CC8**, **0012DD1C**, & **0014C46A** from **Tomb of the Unknown Knight.esp**.' ]
```

---

Yes you are right, I was working off the sunk ships.esp wild edit message and must not have inputted the new form IDs.